packages/react-components/react-tree/stories/D_flatTree/TreeMultiSelection.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```

Comment:
Something very odd is happening with the clicks for me can you repro it? I sometimes need multiple clicks to invoke the selection
![multi click checkbox](https://github.com/microsoft/fluentui/assets/20744592/581c18dd-cd63-4393-ae8f-31565640cd5c)

![multi click radio](https://github.com/microsoft/fluentui/assets/20744592/b089285c-cdc9-4495-a927-8cda4a00207e)

Reply:
Yeah, that's because of the default prevention on `change` event, I'll also have to handle that on a separate PR